service_sysv_linux.go

```diff
@@ -143,6 +143,23 @@ func (s *sysv) Run() (err error) {
 	return s.i.Stop(s)
 }
 
+func (s *sysv) Status() (Status, string, error) {
+	_, out, err := runWithOutput("service", s.Name, "status")
+	if err != nil {
+		return StatusError, out, err
+	}
+
+	if out == "Running" {
```

Comment:
Slight preference to making this a switch, let default case return StatusUnknoown, no function level return.